### ISSUE TITLE
Crash at TypedLocalVariableChecker::filterStatementsVar

### ISSUE THREAD

**Issue by sasezaki** [struggle-for-php/sfp-psalm-typed-local-variable-plugin#9]
```
Scanning files...
Analyzing files...

░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░  60 / 486 (12%)
░░░░░░░░░░░░░░░░░░░░░░░░░Uncaught Exception: Unsupported operand types
Stack trace in the forked worker:
#0 /home/sasezaki/.config/composer/vendor/struggle-for-php/sfp-psalm-typed-local-variable-plugin/src/TypedLocalVariableChecker.php(58): Sfp\Psalm\TypedLocalVariablePlugin\TypedLocalVariableChecker::filterStatementsVar(
```

this issue seems be caused by is_array check missing.

**Comment by sasezaki** [comment]
`Uncaught Exception: Unsupported operand types` problem still exists

**Comment by sasezaki** [comment]
closed with #11
